source/agora/consensus/protocol/Nominator.d

```diff
@@ -98,6 +98,7 @@ public extern (C++) class Nominator : SCPDriver
 
     /// The mapping of all known quorum sets
     private SCPQuorumSetPtr[Hash] known_quorums;
+    private SCPQuorumSetPtr[NodeID] all_quorums;
```

Comment:
Why the new member ? Isn't it replacing `known_quorums` ?

Reply:
Reverted to `known_quorums`.